modules/system/lang/vn/lang.php

```diff
@@ -0,0 +1,434 @@
+<?php
+
+return [
+    'app' => [
+        'name' => 'OctoberCMS',
+        'tagline' => 'Getting back to basics'
+    ],
+    'locale' => [
+        'be' => 'Беларуская',
+        'bg' => 'Български',
+        'cs' => 'Čeština',
+        'da' => 'Dansk',
+        'en' => 'English (United States)',
+        'en-au' => 'English (Australia)',
+        'en-ca' => 'English (Canada)',
+        'en-gb' => 'English (United Kingdom)',
+        'et' => 'Eesti',
+        'de' => 'Deutsch',
+        'el' => 'Ελληνικά',
+        'es' => 'Español',
+        'es-ar' => 'Español (Argentina)',
+        'fa' => 'فارسی',
+        'fr' => 'Français',
+        'fr-ca' => 'Français (Canada)',
+        'hu' => 'Magyar',
+        'id' => 'Bahasa Indonesia',
+        'it' => 'Italiano',
+        'ja' => '日本語',
+        'kr' => '한국어',
+        'lt' => 'Lietuvių',
+        'lv' => 'Latviešu',
+        'nb-no' => 'Norsk (Bokmål)',
+        'nl' => 'Nederlands',
+        'pl' => 'Polskie',
+        'pt-br' => 'Português (Brasil)',
+        'pt-pt' => 'Português (Portugal)',
+        'ro' => 'Română',
+        'ru' => 'Русский',
+        'fi' => 'Suomi',
+        'sv' => 'Svenska',
+        'sk' => 'Slovenský',
+        'tr' => 'Türkçe',
+        'uk' => 'Українська мова',
+        'vn' => 'Tiếng việt',
```

Comment:
You need to include this `'vn' => 'Tiếng việt',` in the system/lang/en/lang.php as well as it's the fallback for any missing translations in every other language file

Reply:
ok, i've update

Reply:
but i dont find the "locale" in some language